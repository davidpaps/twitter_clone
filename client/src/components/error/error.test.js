import React from "react";
import { shallow } from "enzyme";
import Error from "./error";
import renderer from "react-test-renderer";

describe("Error", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<Error />)));

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(1);
  });

  it("should render a <img />", () => {
    expect(wrapper.find("img").length).toEqual(1);
  });

  it("should render correctly", () => {
    const tree = renderer.create(<Error />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
