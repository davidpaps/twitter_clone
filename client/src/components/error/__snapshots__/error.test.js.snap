// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render correctly 1`] = `
<div
  className="Error"
>
  <h1
    className="text-center"
  >
    Error - Page Does Not Exist!
  </h1>
  <img
    alt=""
    src="https://www.oho.com/sites/default/files/styles/width_1160/public/2020-03/shutterstock_1076235776.jpg?itok=gNVLEOJt"
  />
</div>
`;
